vueConfig/mew.js

```diff
@@ -8,7 +8,7 @@ const webpackConfig = {
   },
   devServer: {
     https: true,
-    host: 'localhost',
+    host: '0.0.0.0',
```

Comment:
this needs to be localhost for https to work

Reply:
Fixed. Thank you!